packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -346,6 +347,10 @@ export default class QuanticFacet extends LightningElement {
       this.onSelectClickHandler(item);
     }
     this.clearInput();
+    registerToStore(this.engineId, 'facets', {
+      label: this.label,
+      facetId: this.facetId ?? this.field
```

Comment:
we should just use facetId and make sure this value is defined by setting it in the initialize function.

```
this.facet = CoveoHeadless.buildFacet(engine, {options});
this.facetId = this.facet.facetId;
```

Reply:
same for other facets as well.